Library/Formula/zorba.rb

```diff
@@ -0,0 +1,46 @@
+
+class Zorba < Formula
+  homepage "http://www.zorba.io/"
+  url "https://github.com/28msec/zorba/archive/3.0.tar.gz"
+  sha1 "c444cde689600aab1172b4974348dd6626e92261"
+
+  option "disable-big-integer", "Use 64 bit precision instead of arbitrary precision for performance"
+  option "enable-ssl-verification", "Enable SSL peer certificate verification"
```

Comment:
If it absolutely won't work with the system ` cURL ` or Homebrew's ` cURL ` built against Apple's Security Framework, you'd structure the dependency like ` depends_on "curl" => "with-openssl" `.